big_tests/tests/mam_SUITE.erl

```diff
@@ -3068,9 +3068,11 @@ check_user_exist(Config) ->
   ok = rpc(mim(), ejabberd_auth, try_register, [JID, AdminP]),
   %% admin user already registered
   {ok, HostType} = rpc(mim(), mongoose_domain_core, get_host_type, [AdminS]),
-  true = rpc(mim(), mongoose_users, does_user_exist, [HostType, JID]),
-  false = rpc(mim(), mongoose_users, does_user_exist, [HostType, mongoose_helper:make_jid(<<"fake-user">>, AdminS)]),
-  false = rpc(mim(), mongoose_users, does_user_exist, [HostType, mongoose_helper:make_jid(AdminU, <<"fake-domain">>)]),
+  #{result := true} = rpc(mim(), mongoose_hooks, does_user_exist, [HostType, JID, stored]),
```

Comment:
Not sure if this is the best place to add `does_user_exist` hook testing, but I don't have any better suggestions either.